mrtt-ui/src/views/SiteQuestionsOverview/SiteQuestionsOverview.js

```diff
@@ -138,6 +138,13 @@ const SiteOverview = () => {
                 </Link>
               </WideTh>
             </tr>
+            <tr>
+              <WideTh>
+                <Link to={`/sites/${siteId}/form/ecological-status-and-outcomes`}>
+                  {pageLanguage.formName.ecologicalStatusOutcomes}
+                </Link>
+              </WideTh>
+            </tr>
           </tbody>
         </TableAlertnatingRows>
```

Comment:
I dont know if you want to do this here or as part of another ticket, but the site overview has diverged from the mockups regarding what section is under what heading (registration, intervention, monitoring)

<img width="417" alt="Screen Shot 2022-09-14 at 3 46 47 PM" src="https://user-images.githubusercontent.com/1740152/190268719-3ae6277b-c53e-45c1-bd7d-6167db0472ba.png">

Reply:
thanks for pointing this out! will take a closer look and maybe check in with @alanjleonard

Reply:
This will be covered in this ticket https://github.com/globalmangrovewatch/gmw-users/issues/295